internal/storage/migrations/20231122000000_initial_schema.sql

```diff
@@ -0,0 +1,35 @@
+-- +goose Up
+-- create "roles" table
+CREATE TABLE "roles" (
+  "id" character varying NOT NULL,
+  "name" character varying(64) NOT NULL,
```

Comment:
What's the reasoning for setting this limit at the storage level? In my experience, doing this is just setting yourself up for running future database migrations to increase the limit - 64 characters is pretty low for something you intend to expose to users.

Reply:
64 characters seems to be a common limit. This field is part of a unique index and needs to have a limit to ensure performance.